modules/claims_api/app/workers/claims_api/special_issue_updater.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'sentry_logging'
+
+module ClaimsApi
+  class SpecialIssueUpdater
+    include Sidekiq::Worker
+    include SentryLogging
+
+    # Update special issues for a single contention/disability
+    #
+    # @param user [OpenStruct] Veteran to attach special issues to
+    # @param contention_id [Hash(claim_id:, code:, name:)] Identifier to match existing contention
+    # @param special_issues [Array(String)] List of special issues to append
+    def perform(user, contention_id, special_issues)
```

Comment:
concept here is that this logic will be called for each disability provided in the json payload, so:
```
payload[:disabilities].each do |disability|
   ClaimsApi::SpecialIssueUpdater.perform_later("special_issues from both disability and secondary disabilities for that disability")
end
```